### DISCUSSION TITLE
JPEG-Datei in TInvoiceAttachmentTypeHelper.GetTypeFromFilename hinzufügen

### DISCUSSION THREAD

**Discussion by BlueStarHH** [LandrixSoftware/XRechnung-for-Delphi#26] [Ideas]
Bitte in der Unit intf.Invoice in der Funktion 

class function TInvoiceAttachmentTypeHelper.GetTypeFromFilename(
  const _Filename: String): TInvoiceAttachmentType;

Die Zeile 
  if SameText(fileExt,'.jpg') then
ändern in
  if SameText(fileExt,'.jpg') or SameText(fileExt,'.jpeg') then

**Comment by landrix** [comment]
ok
für solche Dinge bitte immer ein Issue eröffnen, keine Diskussion :)

https://github.com/LandrixSoftware/XRechnung-for-Delphi/issues/27

**Comment by landrix** [comment]
erledigt
